src/components/player.js

```diff
@@ -250,13 +250,23 @@ export default class Player extends BaseObject {
     this.core.mediaControl.container.stop();
   }
 
+  
+  /**
+   * seeks the current video (`source`). For example, `player.seek(120)` will seek to second 120 (2minutes) of the current video.
+   * @method seekSeconds
+   * @param {Number} time should be a number between 0 and the video duration.
+   */
+  seek(time) {
+    this.core.mediaControl.container.setCurrentTime(time, true);
+  }
+
   /**
    * seeks the current video (`source`). For example, `player.seek(50)` will seek to the middle of the current video.
    * @method seek
    * @param {Number} time should be a number between 0 and 100.
    */
-  seek(time) {
-    this.core.mediaControl.container.setCurrentTime(time);
+  seekPercentage(percentage) {
+    this.core.mediaControl.container.setCurrentTime(percentage);
```

Comment:
I agree with you, but I don't think this is that critical, we would have two methods that are almost the same both needs to dispatch the CONTAINER_EVENT so instead of this: 
```
setCurrentTime(time, inseconds) {
    this.trigger(Events.CONTAINER_SEEK, time, this.name);
    if (inseconds) {
      this.playback.seek(time);
    } else {
      this.playback.seekPercentage(time);
    }
  }
```
So we would have something like this:
```
setCurrentTime(time) {
    this.trigger(Events.CONTAINER_SEEK, time, this.name);
    this.playback.seek(time);
  }

setCurrentPercentage(percentage) {
    this.trigger(Events.CONTAINER_SEEK, percentage, this.name);
    this.playback.seekPercentage(percentage);
}
```